Formula/parsnp.rb

```diff
@@ -13,18 +14,22 @@ class Parsnp < Formula
     sha256 "69d7edf2c58a0ee90edd73c653090b578056a36ef0d939ad065ea0279eb7886e" => :x86_64_linux
   end
 
-  fails_with :clang # needs openmp
-
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
 
   depends_on "fasttree"
-  depends_on "gcc" if OS.mac? # needs opemmp
   depends_on "harvest-tools"
   depends_on "libmuscle"
-  depends_on "zlib" unless OS.mac?
+
+  if OS.mac?
+    depends_on "gcc"
```

Comment:
I like adding `depends_on "gcc" # needs opemmp` here as well. The other common reason to depend on `gcc` is for Fortran.